biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.tsx

```diff
@@ -17,13 +17,70 @@ import SimulationBoard, {
   GridItemRet,
   GridItemLines,
 } from '../../components/simulation-board';
+import PacmanLoader from 'react-spinners/PacmanLoader';
+
+const SimBoardPlayable = (props: {
+  simulationResults: BiogridSimulationResults;
+  stateFrame: number;
+}) => {
+  const stateToGridItemRet = (state: any): GridItemRet[] => {
+    return state.nodes.map((node: any) => {
+      return {
+        gridItemName: node.value.gridItemName,
+        relativePosition: node.value.position ||
+          node.value.relativePosition || { x: 0, y: 0 },
+      };
+    });
+  };
+  const stateToGridItemLines = (state: any): GridItemLines[] => {
+    return state.edges.map((edge: any) => {
+      return {
+        fromItem: edge.v,
+        toItem: edge.w,
+        powerThroughLinesKiloWatts: edge.value.power,
+      };
+    });
+  };
+  return (
+    <>
+      <div>
+        Time of day: {props.stateFrame % 12 === 0 ? 12 : props.stateFrame % 12}
+        :00 {props.stateFrame >= 12 ? 'PM' : 'AM'}
+      </div>
+      <SimulationBoard
+        grid_height_km={props.simulationResults.townSize.height}
+        grid_width_km={props.simulationResults.townSize.width}
+        // TODO add changing indices to show the progression of time for each subsequent state
+        // Find the GitHub issue: https://github.com/googleinterns/step141-2020/issues/64
+        items={stateToGridItemRet(
+          props.simulationResults.states[
+            props.stateFrame < props.simulationResults.states.length
+              ? props.stateFrame
+              : 0
+          ]
+        )}
+        lines={stateToGridItemLines(
+          props.simulationResults.states[
```

Comment:
all simulations start at midnight, and go 6-8 hours without any arrows because there's no sunlight yet. Because of that when you click "play" it's not immediately obvious the simulation is running. Can we add a todo or add in this PR a change that shows some text like "no sunlight" if there's no arrows displayed in the sim for that hour

Reply:
Added the TODO https://github.com/googleinterns/step141-2020/issues/90